src/screens/HomeScreen/HomeScreen.tsx

```diff
@@ -28,12 +31,41 @@ const HomeScreen = observer((_props: HomeScreenProps) => {
     });
   };
 
+  const getEvents = (_location?: string) => {
+    getItemsFromAsyncStorage('RootStore').then(res => {
```

Comment:
Why fetch from async store? Check if events array length from model is greater than 0

Reply:
Fixed.